src/m_misc.c

```diff
@@ -450,6 +451,20 @@ default_t defaults[] = {
     "1 to enable \"ghost monsters\" (resurrected pools of gore are translucent)"
   },
 
+  {
+    "blockmapfix",
+    (config_t *) &blockmapfix, NULL,
+    {0}, {0,1}, number, ss_enem, wad_no,
+    "1 to enable blockmap bug fix"
+  },
+
+  {
+    "pistolstart",
```

Comment:
Should we get rid of the command line argument then?

Reply:
Sorry just passing by but I think it should stay. It has become a bit of a standard command line argument (Woof, Pr+UM and DSDA right?). For my uses case I like to set it in my launch scripts for community projects or replays of wads I'm getting familiar with etc but otherwise leave it off for new/tough uniform projects.

Reply:
> Sorry just passing by but I think it should stay. It has become a bit of a standard command line argument (Woof, Pr+UM and DSDA right?). 

Then we'll need two variables, one tracking the command line argument and one tracking the menu item. Else setting the feature on the command line will set and override the menu item each time.

Reply:
Agreed, `-pistolstart` is now the standard parameter.